gatekeeper/datadog_checks/gatekeeper/metrics.py

```diff
@@ -3,6 +3,8 @@
     'gatekeeper_constraint_templates': 'constraint_templates',
     'gatekeeper_constraint_template_ingestion_count': 'constraint_template_ingestion.count',
     'gatekeeper_constraint_template_ingestion_duration_seconds': 'constraint_template_ingestion.duration.seconds',
+    'gatekeeper_validation_request_count': 'validation.request.count',
+    'gatekeeper_validation_request_duration_seconds': 'validation.request.duration.seconds',
```

Comment:
Shall we take the opportunity to also get the `mutation_request_count` and `mutation_request_duration_seconds` ones? The deprecation of the previous one seems a consequence of the new mutating webhook.

Reply:
Yup! I'll add the new mutation metrics